.gitignore

```diff
@@ -2,3 +2,4 @@
 tmp
 .bundle
 vendor
+**/*.gem
```

Comment:
you can put *.gem into your global .gitignore

https://github.com/grosser/dotfiles/blob/master/gitconfig#L61-L62